org-code-javabuilder/lib/src/test/java/org/code/neighborhood/GridTest.java

```diff
@@ -0,0 +1,60 @@
+package org.code.neighborhood;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+public class GridTest {
+
+  @Test
+  void constructorBuildsGrid() {
+    GridSquare s = new GridSquare(1);
+    GridSquare[][] squares = new GridSquare[1][1];
+    squares[0][0] = s;
+    Grid grid = new Grid(squares);
+    assertTrue(grid.validLocation(0, 0));
+    assertFalse(grid.validLocation(0, 1));
+    assertFalse(grid.validLocation(1, 0));
+  }
+
+  @Test
+  void validLocationReturnsFalseForInvalidLocations() {
+    GridSquare s = new GridSquare(1);
+    GridSquare[][] squares = new GridSquare[1][1];
+    squares[0][0] = s;
+    Grid grid = new Grid(squares);
+    assertTrue(grid.validLocation(0, 0));
```

Comment:
NIT: This assertTrue should probably be in its own test as it doesn't match the goal of this test.